ddtrace/_trace/tracer.py

```diff
@@ -332,6 +342,9 @@ def sampler(self, value):
     https://ddtrace.readthedocs.io/en/stable/configuration.html#DD_TRACE_SAMPLING_RULES""",
             category=DDTraceDeprecationWarning,
         )
+        if self._apm_opt_out:
+            log.error("Cannot set a custom sampler with Standalone ASM mode")
```

Comment:
isn't a warning enough? Application is not crashing here.

Reply:
yeah, true